tests/proofs/lqt/lqt-spec.md

```diff
@@ -93,3 +93,140 @@ module LQT-TOKEN-GETALLOWANCE-SPEC
     requires notBool(Amount ==Int 0)
 endmodule
 ```
+
+```k
+module LQT-TOKEN-MINTORBURN-SPEC
+  imports LQT-TOKEN-VERIFICATION
+
+  claim <k> #runProof(MintOrBurnParams(Quantity, Address)) => .K ... </k>
+        <stack> .Stack </stack>
+        <tokens> Tokens => #incrementTokens(Tokens, Address, Quantity) </tokens>
+        <myamount> #Mutez(Amount) </myamount>
+        <adminAddress> Admin </adminAddress>
+        <senderaddr> Sender </senderaddr>
+        <totalSupply> TotalSupply => absInt(TotalSupply +Int Quantity) </totalSupply>
+        <operations> _ => .InternalList </operations>
+    requires Amount ==Int 0
+     andBool Sender ==K Admin
+     andBool #tokensFor(Tokens, Address) +Int Quantity >=Int 0
+     andBool Address in_keys(Tokens)
+
+   claim <k> #runProof(MintOrBurnParams(Quantity, Address)) => Aborted(?_, ?_, ?_, ?_) ... </k>
+        <stack> .Stack => ?_:FailedStack </stack>
+        <tokens> Tokens </tokens>
+        <myamount> #Mutez(Amount) </myamount>
+        <adminAddress> Admin </adminAddress>
+        <senderaddr> Sender </senderaddr>
+    requires Address in_keys(Tokens)
+     andBool notBool( Amount ==Int 0
+              andBool Sender ==K Admin
+              andBool #tokensFor(Tokens, Address) +Int Quantity >=Int 0
```

Comment:
If

```
Address in_keys(Tokens)
```

Then only case can apply in the following function:

```
#tokensFor(Tokens, Address)
```

But the contract code only checks this condition, as far as I can tell:

```
#tokensFor(Tokens, Address) +Int Quantity >=Int 0
```

Thus, I think that there is a missing case in both the positive and negative cases.